src/braket_container.py

```diff
@@ -149,14 +150,27 @@ def kick_off_customer_script(entry_point : str) -> multiprocessing.Process:
         str_module, _, str_method = entry_point.partition(":")
         customer_module = importlib.import_module(str_module)
         customer_method = getattr(customer_module, str_method)
-        customer_code_process = multiprocessing.Process(target=customer_method)
+
+        process_kwargs = {"target": customer_method}
+
+        hp_file = os.environ["AMZN_BRAKET_HP_FILE"]
```

Comment:
Is this the same as SM_HPS ?

Reply:
It looks like it's similar, but SM_HPS is a string. Is that a stable API we can rely on? I believe AMZN_BRAKET_HP_FILE is the documented way of getting hyperparameters for us, but SM_HPS may be slightly more efficient

Reply:
Doc link: https://docs.aws.amazon.com/braket/latest/developerguide/braket-jobs-inputs-and-outputs.html#braket-jobs-environmental-variables